### ISSUE TITLE
Counters

### ISSUE THREAD

**Issue by orightythen1** [Tucsky/aggr#20]
Hi - great work, really appreciate this project.

Just curious whether there was an easier way to create counters, in the last version the counter you created was automatically associated to the trades it was tracking. It seems now on each individual counter you have to add all the exchanges to each one? 

Is there a faster solution at all?

Thanks.

**Comment by Tucsky** [comment]
On v2, there was a list of connected pairs the app was listening to, the counter module was automatically connected to those pairs. 

On v3 the app also have that list of connected pairs (across all panes), when the counter pane is added, all pairs currently connected to the app are assigned to the pane
Looks like same behavior to me... ?


Now if you add multiple counters pane, you are going to need to choose different markets, for each panes, of course. Pretty much the same as V2 where you had to open multiple app, select the pair(s) on each and enable counter feature.

I know the search market have room for improvements, I talked about it in #17
- market categories (spot, perp, futures, usd perp, btc perp etc)
- custom groups that the user could save and reuse for faster market selection

> Is there a faster solution at all?

Not that I'm aware of

If you have any suggestions tho, I'm listening.

**Comment by orightythen1** [comment]
Thank you for the clarification - I'm not used to having multiple panes on the same window/browser.

In v2 I had firefox btc.aggr for perps, brave for spot btc.aggr and again firefox for eth.aggr (I assume this created them as separate entities so when I added counters, it was for each group of exchanges rather than combining them all). Using different browsers allowed me to keep them separate, and ETH had its own url which I assume had the same effect. 

In the new version could I achieve this by creating profiles for each spot/perps/eth and then when adding the counters it will be for the listed exchanges?  One issue I'm having is in v2 the pop up mode was nice and tight, so I could open my 3 versions and line them up clean. I don't appear to be able to close the window in on the 1 pane to make it compact like v2. I took a screenshot: http://prntscr.com/15lsz00 

Appreciate the information.

**Comment by Tucsky** [comment]
Okay so I tried to make a screen showing counter + trades for the following market group
- BTC SPOT
- BTC PERP
- ETH SPOT
- ETH PERP

It showed to me that the UI really lack some kind of shortcut to create a pane using same markets as some other pane. First had to create all the trades pane with specific markets, and do it all over again for the counters.
also in the search window, the condition for showing the connect / disconnect button is bugged, I just noticed as I forgot to add bitmex:xbtusd on some pane, but the market was already connected to the app and selecting it from the list on the left wouldn't show the "connect" button at the bottom despite adding it to the list on the right. Anyway I ended up editing the json file lol

![image](https://user-images.githubusercontent.com/5738336/122236500-dbb34880-cebe-11eb-9740-0ac635890d00.png)

[link to the workspace](https://pastebin.com/3Czjen75)
Open main settings sidebar and drop it into the app to add

Will fix thoses issue asap as well some more options on the search window to auto select all markets from another pane
